src/Analyzers/Core/CodeFixes/RemoveUnusedParametersAndValues/AbstractRemoveUnusedValuesCodeFixProvider.cs

```diff
@@ -62,8 +60,9 @@ internal abstract class AbstractRemoveUnusedValuesCodeFixProvider<TExpressionSyn
         private static readonly SyntaxAnnotation s_unusedLocalDeclarationAnnotation = new();
         private static readonly SyntaxAnnotation s_existingLocalDeclarationWithoutInitializerAnnotation = new();
 
-        public sealed override ImmutableArray<string> FixableDiagnosticIds => ImmutableArray.Create(IDEDiagnosticIds.ExpressionValueIsUnusedDiagnosticId,
-                                                                                                    IDEDiagnosticIds.ValueAssignedIsUnusedDiagnosticId);
+        public sealed override ImmutableArray<string> FixableDiagnosticIds
+            => ImmutableArray.Create(IDEDiagnosticIds.ExpressionValueIsUnusedDiagnosticId,
+                                     IDEDiagnosticIds.ValueAssignedIsUnusedDiagnosticId);
 
```

Comment:
I personally prefer `{ get; } =` to guarantee a one-time only array allocation. Though I don't know if `FixableDiagnosticIds` is called multiple times.